max_xy: 10.0 # [m]
min_xy: -10.0 # [m]

min_z: 4.0 # [m]
max_z: 6.0 # [m]

max_heading: 3.14 # [rad]
min_heading: -3.14 # [rad]

goto_relative_altitude_up: 1.0 # [m]
goto_relative_altitude_down: 1.0 # [m]

test_line_tracker: false
test_mpc_gotos: true

trajectory_dt: 0.1 # [s]

line_trajectory:

  p1: -5.0 # [m]
  p2: 5.0 # [m]

  speed: 3.0 # [m/s]

  heading_rate: 1.0 # [rad/s]

looping_circle_trajectory:

  radius: 15.0 # [m]
  speed: 8.0 # [m/s]
  duration: 30.0 # [s]

switch_estimator_after_takeoff:
  active: false
  target_estimator: "GPS"
